src/transport/reqwest.rs

```diff
@@ -56,7 +56,7 @@ pub struct TransportReqwest {
 impl Transport for TransportReqwest {
     async fn send(&self, request: TransportRequest) -> Result<TransportResponse, PubNubError> {
         let request_url = prepare_url(&self.hostname, &request.path, &request.query_parameters);
-        info!("{}", request_url);
+        info!("{} {:?} {}", request.method, request.headers, request_url);
```

Comment:
Imo it would be nice to add some more information about where is that log generated: e.g. 
```suggestion
        info!("Sending data to pubnub: {} {:?} {}", request.method, request.headers, request_url);
```